app/models/agents/http_agent.rb

```diff
@@ -1,6 +1,6 @@
 module Agents
 
-  class HttpStatusAgent < Agent
+  class HttpAgent < Agent
```

Comment:
You have to be careful when renaming an Agent because it could be referenced in exported Scenarios.  If possible, I'd suggest not renaming it.